3_acm/config-management.yaml.tpl

```diff
@@ -3,8 +3,11 @@ kind: ConfigManagement
 metadata:
   name: config-management
 spec:
+  channel: cs
```

Comment:
Do you know why the `channel` field is not used before?

Reply:
This is a new field for the channel "config-sync".

Reply:
I see.